File: Tareas/T10SerieA.rb
=begin
Nombre: OSCAR IGNACIO GUAJARDO HERNANDEZ
Matricula:1865021
Hora: V4-6 Martes
=end
j=0
mensaje = ""
puts 'BIENVENIDO A LA SERIE A'
puts 'Ingrese hasta que posicion desea llegar en la serie:'
n = gets().to_i 
n=n+1
i=1
for i in (1...n)
  j = (2*i)-1
  num = i ** j
  mensaje ="#{mensaje} #{num}, "
end
puts mensaje

File: Tareas/T10SerieA.java
/*  NOMBRE : OSCAR IGNACIO GUAJARDO HERNANDEZ
MATRICULA : 1865021  HORA: V4-6;*/

import javax.swing.JOptionPane;

public class T10SerieA
{
	public static void main (String[] arg)
	{
		String mensaje="";
		int n=0, i=0, j=0;
		double num=0;
		String Ventana = JOptionPane.showInputDialog("BIENVENIDO A LA SERIE A\n:"
				+"Ingrese hasta que posicion desea llegar en la serie: ");
	
		n = Integer.parseInt(Ventana);
		for(i=1;i<=n;i++)
		 {    
			j=2*i-1;
			num= Math.pow(i,j);
			mensaje=mensaje+num+", ";
		 }
		 JOptionPane.showMessageDialog(null,mensaje);
		 
	}
}
